libs/sdk-ui-dashboard/src/presentation/topBar/menuButton/DefaultMenuButton.tsx

```diff
@@ -26,24 +28,151 @@ const bubbleAlignPoints: IAlignPoint[] = [{ align: "cl tr" }];
 export const DefaultMenuButton = (props: IMenuButtonProps): JSX.Element | null => {
     const { menuItems } = props;
     const [isOpen, setIsOpen] = useState(false);
+    const [autofocusSubmenu, setAutofocusSubmenu] = useState(false);
```

Comment:
Maybe call it `autoFocusSubmenu`?

Reply:
imho autofocus as single word is correct, at least consistent with html attribute